PFR/Mathlib/Probability/ConditionalProbability.lean

```diff
@@ -10,6 +10,8 @@ namespace ProbabilityTheory
 open MeasureTheory Set
 open scoped BigOperators
 
+/- [TODO]: `sum_meas_smul_cond_fiber'` should substitute `sum_meas_smul_cond_fiber` in Mathlib once
+the definition of `FiniteRange` is in Mathlib.-/
```

Comment:
Note: I don't think we'll ever put `FiniteRange` in mathlib. @RemyDegenne can confirm/infirm, but I believe the plan is to directly shoot for the stars and get the integral version of entropy.

Reply:
I don't know. I am not sure there is a plan :)